src/components/search-wizard/filters/Statewide.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import stateOfUtahJson from '../../../data/state-of-utah.json';
+
+export default function Statewide({ send }) {
+  useEffect(() => {
```

Comment:
Because I don't need it to fire on each render. This component is kind on contrived. But I wanted to be able to treat all of the filters the same way in AdvancedFilter.